### PR TITLE
fix kill kernel button

### PR DESCRIPTION
**Pull request by jonathanindig**

### CODE REVIEW

File: polynote-frontend/polynote/ui/component/about.ts [hunk 1]
```diff
@@ -185,12 +185,14 @@ export class About extends FullScreenModal {
                         ]);
                         const actionsEl = div([], [
                             iconButton(['start'], 'Start kernel', 'power-off', 'Start').click(() => {
-                                this.publish(new KernelCommand(path, 'kill'));
+                                this.publish(new KernelCommand(path, 'start'));
```
Comment (jonathanindig) [inline polynote-frontend/polynote/ui/component/about.ts]: 🤦‍♂

File: polynote-frontend/polynote/ui/component/about.ts [hunk 2]
```diff
@@ -185,12 +185,14 @@ export class About extends FullScreenModal {
...
                                 getKernelStatuses();
+                                setTimeout(() => getKernelStatuses(), 5000); // TODO: this should be event-driven!
```
Comment (jonathanindig) [inline polynote-frontend/polynote/ui/component/about.ts]: this `setTimeout` hack is annoying. We should revisit this when we get to the UI rethink we talked about a while ago

File: polynote-frontend/polynote/ui/component/about.ts (hunk 1 above)
Comment (jeremyrsmith) [reply polynote-frontend/polynote/ui/component/about.ts]: Guessing this is my doing 😛

Comment (jonathanindig) [reply polynote-frontend/polynote/ui/component/about.ts]: Nah this was all me: https://github.com/polynote/polynote/pull/485/files#diff-36c99d07368a1579806b1d11c5d9f8ed 😅

**Pull request merged**